exercises/react/react.spec.js

```diff
@@ -97,7 +97,25 @@ describe('React module', () => {
     inputCell.setValue(4);
     expect(callback.values).toEqual([222]);
   });
+  
+  xtest('static callbacks fire even if their own value has not changed', () => {
+    const inputCell = new InputCell(1);
+    const output = new ComputeCell(
+      [inputCell],
+      inputs => (inputs[0].value < 3 ? 111 : 222),
+    );
+
+    const callback = new CallbackCell(cell => "cell changed");
```

Comment:
```suggestion
    const callback = new CallbackCell(() => "cell changed");
```

Or if you want to indicate there is an argument passed in:

```suggestion
    const callback = new CallbackCell((_cell) => "cell changed");
```

Reply:
Oh right. We changed this rule back to the default. Try `(ignore)`.

- Run the linter locally! (See https://github.com/exercism/javascript/blob/master/scripts/lint)
- You can pass in an assignment (IIRC `ASSIGNMENT=react`)

Reply:
@yyyc514

Reply:
Will try that later today I hope.  Thanks.